tests/unit/checks/generictests/datasets/oracle_jobs_regression.py

```diff
@@ -0,0 +1,53 @@
+# -*- encoding: utf-8
+# yapf: disable
+checkname = 'oracle_jobs'
+
+info = [
+    [
```

Comment:
add more example outputs for ALL formats:
- len(line) < 3,
- 3 <= len(line) <= 10,
- len(line) > 10